### PR TITLE
add wang ergo keyboard

### PR DESCRIPTION
**Pull request by oceeean**
## Description

add wang ergo keyboard to qmk and via support
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by oceeean** [comment]
i dont know why `keyboards/checkerboards/quark_squared/info.json` is in my PR

**Review by drashna — APPROVED** [review]

File: keyboards/ocean/wang_ergo/readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Wang Ergo
+
+![Wang ergo](https://i.imgur.com/vsYWYfx.jpeg)
+
```
Comment (fauxpark) [inline keyboards/ocean/wang_ergo/readme.md:4]:
```suggestion
![Wang ergo](https://i.imgur.com/vsYWYfxh.jpeg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by fauxpark** [comment]
The changes to `keyboards/checkerboards/quark_squared/info.json` should be removed from this PR; the problem with that file (incorrect line endings) has already been fixed.

File: keyboards/checkerboards/quark_squared/info.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-        }
-    }
+{
```
Comment (drashna) [inline keyboards/checkerboards/quark_squared/info.json:1]: Please revert this chnage.  

You may need to rebase.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/checkerboards/quark_squared/info.json (hunk 2 above)
Comment (oceeean) [reply keyboards/checkerboards/quark_squared/info.json:1]: how do i revert this

**Comment by drashna** [comment]
You have some merge conflicts here.

**Comment by oceeean** [comment]
> You have some merge conflicts here.

can i just close this PR and open a new one?

**Pull request closed without merging**